Classes/Service/EmailService.php

```diff
@@ -12,23 +12,32 @@
 namespace JWeiland\Telephonedirectory\Service;
 
 use JWeiland\Telephonedirectory\Configuration\ExtConf;
+use JWeiland\Telephonedirectory\Domain\Model\Employee;
 use TYPO3\CMS\Core\Mail\MailMessage;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder;
+use TYPO3\CMS\Extbase\Security\Cryptography\HashService;
 use TYPO3\CMS\Extbase\Utility\LocalizationUtility;
+use TYPO3\CMS\Fluid\View\StandaloneView;
 
 /**
  * Send an information mail to employees email address
  */
 class EmailService
 {
-    /**
-     * @var ExtConf
-     */
-    protected $extConf;
+    protected ExtConf $extConf;
+    protected UriBuilder $uriBuilder;
+    protected array $settings = [];
 
-    public function __construct(ExtConf $extConf)
-    {
+    public function __construct(
+        ExtConf $extConf,
```

Comment:
OK for now, but in future we should try to prevent loading stateful objects in stateless objects. Yes, just learned at DevDays. I will talk to you about that later.